client_scrapers/scrape_stats.php

```diff
@@ -45,7 +51,9 @@
 				$autoRun = FALSE;
 			}
 		}
-		if (empty($profileIDs)){die("Please specify at least 1 profile ID! Usage: scrape_stats.php [-auto] 00000000");}
+		if($USBProfile){
+			if (empty($profileIDs)){die("Please specify at least 1 profile ID! Usage: scrape_stats.php [-auto] 00000000");}
```

Comment:
Wouldn't this always fail since for USB profiles you specify the directory instead of the profileID?